Client/Frontend/Browser/BackForwardTableViewCell.swift

```diff
@@ -0,0 +1,149 @@
+//
+//  BackForwardTableViewCell.swift
+//  Client
+//
+//  Created by Tyler Lacroix on 5/17/16.
+//  Copyright © 2016 Mozilla. All rights reserved.
+//
+
+import UIKit
+import Storage
+
+class BackForwardTableViewCell: UITableViewCell {
+    
+    struct BackForwardViewCellUX {
+        static let bgColor = UIColor.init(colorLiteralRed: 0.7, green: 0.7, blue: 0.7, alpha: 1)
+        static let faviconWidth = 20
+        static let faviconPadding:CGFloat = 20
+        static let labelPadding = 20
+        static let borderSmall = 2
+        static let borderBold = 5
+        static let fontSize:CGFloat = 12.0
+    }
+    
+    var faviconView: UIImageView!
+    var label: UILabel!
+    var bg: UIView!
+    
+    var connectingForwards = true
+    var connectingBackwards = true
+    
+    var currentTab = false  {
```

Comment:
I would rename this to `isCurrentTab` to help indicate it's a boolean since we use `currentTab` across the app to store the current tab instance.